html/PhotoListGallery.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>List of Person Details</title>
+		<style type="text/css">
+			
+			body{
+				width: 960px;
+				padding-left: 450px;
+			}
+			.head {
+				text-align: center;
+				color: teal;
+				text-decoration: underline;
+			}
+
+			#box {
+				border: 1px;
+				border-style: solid;
+				border-color: orange;
+				width: 960px;
+				padding-left: 0px;
+			}
+
+			img {
+				width: 200px;
+				height: 200px;
+			}
+
+			#left {
+				display: inline-block;
+				list-style-type: none;	
+				width: 200px;
+				margin-bottom: 50px;
+				float: left;
+			}
+			#center {
+				display: inline-block;
+				list-style-type: none;	
+				width: 200px;
+				margin: 0 180px 50px 180px;
+			}
+			#right {
+				display: inline-block;
+				list-style-type: none;	
+				width: 200px;
+				margin-bottom: 50px;
+				float: right;
+				
+			}
+		</style>
+	</head>
+	<body>
+		<h1 class="head">List of Person Details</h1>
+			<ul id="box">
+				<li id="left">
+					<img src="http://www.v2solutions.com/assets/images/content/vijay.jpg" alt="image not available">
+					Name: Vijay Shah
+					Description: President and CEO, V2 Solutions
+				</li>
+			
+				<li id="center">
+					<img src="http://bordoisila.in/images/28-07-2015-1438083402a.jpg" alt="image not available">
+					Name: Dr. APJ Abdul Kalam
+					Description: Ex-President of India,
+					Scientist at ISRO.
+				</li>
+			
+				<li id="right">
+					<img src="http://im.rediff.com/money/2008/sep/10slid2.jpg" alt="image not available">
+					Name: Ratan Naval Tata
+					Description: CEO of TATA groups
+				</li>
+
+				<li id="left">
+					<img src="http://www.siliconindia.com:81/news/newsimages/special/BOu6f9Y4.jpeg" alt="image not available">
+					Name: Aziz Hashim Premji.
+					Description: Chairman of Wipro Ltd.
+				</li>
+				
+				<li id="center">
+					<img src="http://www.siliconindia.com:81/news/newsimages/special/g94csyBw.jpeg" alt="image not available">
+					Name: N.R. Narayana Murthy.
+					Description: IT industrialist and co-founder of infosys.
+				</li>
+					
+				<li id="right">
+					<img src="http://www.siliconindia.com:81/news/newsimages/special/1Xb5vnME.jpeg" alt="image not available">
+					Name: Lakshmi Niwas Mittal
+					Description: chairman and CEO of ArcelorMittal.
+				</li>
+				
+				<li id="left">
+					<img src="http://www.ses-tek.com/wp-content/uploads/2014/06/Blank-Face-Person-Icon.png" alt="image not available">
+					Name: Aziz Hashim Premji.
+					Description: Chairman of Wipro Ltd.
+				</li>
+				
+				<li id="center">
+					<img src="http://www.ses-tek.com/wp-content/uploads/2014/06/Blank-Face-Person-Icon.png" alt="image not available">
+					Name: N.R. Narayana Murthy.
+					Description: IT industrialist and co-founder of infosys.
+				</li>
+					
+				<li id="right">
```

Comment:
In an html document id attributes value has to be unique. In this case it is not.  Fix it.